rag_experiment_accelerator/evaluation/eval.py

```diff
@@ -774,7 +774,7 @@ def evaluate_prompts(
         os.path.join(config.EVAL_DATA_LOCATION, f"sum_{name_suffix}.csv")
     )
     draw_hist_df(sum_df, run_id, client)
-    generate_metrics(config.INDEX_NAME_PREFIX, run_id, client)
+    generate_metrics(config.EXPERIMENT_NAME, run_id, client)
```

Comment:
What's the reason for this change?
Is it related to the shortened dimensions, or something else?

Reply:
Is there a bug related to this change?

Reply:
This is for an unrelated issue I had when running the process with my config.

The experiment name is [set like this in `evaluation.py`](https://github.com/microsoft/rag-experiment-accelerator/blob/477c61a195b109fee9c1ea81c67200015ebcba7c/rag_experiment_accelerator/run/evaluation.py#L60):

```
    mlflow.set_experiment(config.EXPERIMENT_NAME)
```

And later the metrics complained that the experiment did not exist [here in `eval.py`](https://github.com/microsoft/rag-experiment-accelerator/blob/477c61a195b109fee9c1ea81c67200015ebcba7c/rag_experiment_accelerator/evaluation/eval.py#L377), I believe because `INDEX_NAME_PREFIX` was used instead of `EXPERIMENT_NAME`.

Reply:
> Is there a bug related to this change?

I can't see one, no. I can remove this commit until the issue is confirmed.

Reply:
It's probably fine. I wanted to check that fixing this wouldn't cause it's own issues (e.g. that the change in name wouldn't have a back-compat issue for someone using it today). I also tend to think that it's better to avoid too many "while I'm here" bug fixes, unless they're already logged somewhere. 

That said, if @ritesh-modi is happy with this, I'm also happy. :)

Reply:
To clarify, here is the output of the final step, using the CI configuration, on the development branch:

```
vscode ➜ /workspaces/rag-experiment-accelerator (development) $ python 04_evaluation.py --data_dir='data-ci' --config_path=.github/workflows/config.json
2024/04/26 09:19:22 INFO mlflow.tracking.fluent: Experiment with name 'baseline' does not exist. Creating a new experiment.
2024-04-26 09:19:24,632 - INFO - rag_experiment_accelerator.run.evaluation - Evaluating Index: ci_p-0_cs-1000_o-200_efc-400_efs-400_sp-0_t-1_s-1_oc-0_all-mpnet-base-v2
2024-04-26 09:19:24,643 - INFO - rag_experiment_accelerator.artifact.handlers.artifact_handler - Loading artifacts from path: .github/workflows/artifacts/query_data/eval_output_ci_p-0_cs-1000_o-200_efc-400_efs-400_sp-0_t-1_s-1_oc-0_all-mpnet-base-v2_baseline_baseline_job.jsonl
2024-04-26 09:19:24,645 - INFO - rag_experiment_accelerator.artifact.handlers.artifact_handler - Loaded 6 artifacts from path: .github/workflows/artifacts/query_data/eval_output_ci_p-0_cs-1000_o-200_efc-400_efs-400_sp-0_t-1_s-1_oc-0_all-mpnet-base-v2_baseline_baseline_job.jsonl
Traceback (most recent call last):
  File "/workspaces/rag-experiment-accelerator/04_evaluation.py", line 35, in <module>
    run(
  File "/workspaces/rag-experiment-accelerator/rag_experiment_accelerator/run/evaluation.py", line 36, in run
    eval.evaluate_prompts(
  File "/workspaces/rag-experiment-accelerator/rag_experiment_accelerator/evaluation/eval.py", line 777, in evaluate_prompts
    generate_metrics(config.INDEX_NAME_PREFIX, run_id, client)
  File "/workspaces/rag-experiment-accelerator/rag_experiment_accelerator/evaluation/eval.py", line 377, in generate_metrics
    experiment = dict(client.get_experiment_by_name(experiment_name))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not iterable
```